src/backend/partaj/core/models/referral_activity.py

```diff
@@ -70,6 +71,13 @@ class ReferralActivity(models.Model):
         related_query_name="activity",
     )
 
+    generic_message = models.TextField(
```

Comment:
```suggestion
    message = models.TextField(
```

I think it should just be called message. Yes the field is generically available for any kind of activity but for a given activity instance it's just the "message" for it.